src/licensedcode/data/licenses/osetpl-2.1.yml

```diff
@@ -0,0 +1,7 @@
+key: osetpl-2.1
+short_name: OSET 2.1
+name: OSET Public License v2.1
+
+category: Proprietary Free
```

Comment:
I  wonder if this is a proprietary license or a plain FOSS. This is after all derived from the MPL 2.0. I guess I could reach out to the author...